code/components/extra-natives-five/src/VisualSettingsNatives.cpp

```diff
@@ -127,18 +146,18 @@ static InitFunction initFunction([]()
 
 static HookFunction hookFunction([]()
 {
-#ifdef GTA_FIVE
-	g_visualSettings = hook::get_address<VisualSettingsData*>(hook::get_pattern("48 83 25 ? ? ? ? 00 48 8D ? ? ? ? ? E8 ? ? ? ? E8", 11));
-#elif IS_RDR3
-	g_visualSettings = hook::get_address<VisualSettingsData*>(hook::get_pattern("48 89 2D ? ? ? ? E8 ? ? ? ? E8 ? ? ? ? 48 8D 0D", -11));
-#endif
+	#ifdef GTA_FIVE
+		g_visualSettings = hook::get_address<VisualSettingsData*>(hook::get_pattern("48 83 25 ? ? ? ? 00 48 8D ? ? ? ? ? E8 ? ? ? ? E8", 11));
+	#elif IS_RDR3
+		g_visualSettings = hook::get_address<VisualSettingsData*>(hook::get_pattern("48 89 2D ? ? ? ? E8 ? ? ? ? E8 ? ? ? ? 48 8D 0D", -11));
+	#endif
```

Comment:
Why did you indent the preprocessor directives? These should be aligned to the left to follow standard conventions

Reply:
readability. i've thought there's no standard conventions for preprocessor directives and it's just about personal taste. if i get a review from the maintainers that says that they need to be indented to the left, i'll do them of course. thanks for pointing it out!

Reply:
yeah, you're right, they're indented to the left in the entire workspace.

Reply:
yup, i've squashed them. thanks again for pointing it out!